tests/app/main/helpers/test_frameworks.py

```diff
@@ -364,3 +369,148 @@ def test_check_agreement_is_related_to_supplier_framework_or_abort_does_not_abor
     supplier_framework = {"supplierId": 212, "frameworkSlug": 'g-cloud-8'}
     agreement = {"supplierId": 212, "frameworkSlug": 'g-cloud-8'}
     check_agreement_is_related_to_supplier_framework_or_abort(agreement, supplier_framework)
+
+
+def test_order_frameworks_for_reuse():
+    """Test happy path. Should return 2 frameworks, closest date first."""
+    t09 = '2009-03-03T01:01:01.000000Z'
+    t07 = '2007-03-03T01:01:01.000000Z'
+    t12 = '2012-03-03T01:01:01.000000Z'
+
+    fake_frameworks = [
```

Comment:
These are already ordered in reverse chronological order as they get passed in - might be better to have them mixed up a bit to start with.